libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html

```diff
@@ -1,32 +1,94 @@
-      (click)="onOrganizationClick()"
+      *ngIf="contacts[0].organization?.logoUrl?.href"
+      class="mb-3 flex items-center justify-center border-solid border border-gray-300 rounded-md bg-white h-32"
     >
-      {{ shownOrganization.name }}
+      <gn-ui-thumbnail
+        class="relative h-full w-full"
+        [thumbnailUrl]="contacts[0].organization.logoUrl.href"
+        fit="contain"
+      ></gn-ui-thumbnail>
+    </div>
+    <div class="flex mb-1">
+      <div
+        class="text-primary font-title text-21 mr-2 cursor-pointer hover:underline"
+        (click)="onOrganizationClick()"
+        data-cy="organization-name"
+      >
+        {{ shownOrganization.name }}
+      </div>
+    </div>
+    <div *ngIf="shownOrganization.website" class="mb-6">
+      <a
+        [href]="shownOrganization.website"
+        target="_blank"
+        class="contact-website text-primary text-sm cursor-pointer hover:underline transition-all"
+        >{{ shownOrganization.website }}
+        <mat-icon
+          class="material-symbols-outlined !w-[12px] !h-[12px] !text-[12px] opacity-75 shrink-0"
+          >open_in_new</mat-icon
+        >
+      </a>
+    </div>
+  </div>
+  <div *ngIf="contacts[0].phone" class="mb-5">
+    <div class="flex">
+      <mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+        >call_outline</mat-icon
+      >
+      <div class="flex flex-col ml-2">
+        <p class="text-sm">{{ contacts[0].phone }}</p>
+      </div>
     </div>
   </div>
-  <a
-    *ngIf="contacts.length"
-    [href]="'mailto:' + contacts[0].email"
-    class="text-gray-700 text-sm hover:underline"
-    target="_blank"
-    >{{ contacts[0].email }}</a
-  >
-  <div *ngIf="shownOrganization.website" class="mb-2">
-    <a
-      [href]="shownOrganization.website"
-      target="_blank"
-      class="contact-website text-primary text-sm cursor-pointer hover:underline transition-all"
-      >{{ shownOrganization.website }}
+  <div *ngIf="contacts[0].email" class="mb-5">
+    <div class="flex">
       <mat-icon
-        class="material-symbols-outlined !w-[12px] !h-[12px] !text-[12px] opacity-75"
-        >open_in_new</mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+      >
+        mail_outline</mat-icon
       >
-    </a>
+      <a
+        *ngIf="contacts.length"
+        [href]="'mailto:' + contacts[0].email"
+        class="text-sm hover:underline ml-2"
+        target="_blank"
+        data-cy="contact-email"
+        >{{ contacts[0].email }}</a
+      >
+    </div>
+  </div>
+  <div *ngIf="contacts[0].firstName || contacts[0].lastName" class="mb-5">
+    <div class="flex">
+      <mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+        >person_outline</mat-icon
+      >
+      <div class="flex flex-col ml-2">
+        <p class="text-sm">
+          {{ contacts[0].firstName }} {{ contacts[0].lastName }}
```

Comment:
This might render weirdly if only one of the two is defined

Reply:
I think now it should be handled correctly.